apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/ReguleringService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.etterlatte.tidshendelser
+
+import no.nav.etterlatte.libs.common.rapidsandrivers.lagParMedEventNameKey
+import no.nav.etterlatte.rapidsandrivers.ReguleringEvents
+import no.nav.etterlatte.rapidsandrivers.ReguleringHendelseType
+import no.nav.helse.rapids_rivers.JsonMessage
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.Month
+import java.util.UUID
+
+val year = LocalDate.now().year
+val GRUNNBELOEP_REGULERING_DATO: LocalDate = LocalDate.of(year, Month.MAY, 1)
+
+class ReguleringService(
+    private val rapidsPublisher: (UUID, String) -> Unit,
+) {
+    val logger: Logger = LoggerFactory.getLogger(this::class.java)
+
+    fun execute(jobb: HendelserJobb): List<Long> {
+        logger.info("Handling jobb ${jobb.id}, type ${jobb.type} (${jobb.type.beskrivelse})")
+        when (jobb.type) {
+            JobbType.REGULERING -> startRegulering()
+            JobbType.FINN_SAKER_TIL_REGULERING -> finnSakerTilRegulering()
+            else -> throw IllegalArgumentException("Ikke-støttet jobbtype: ${jobb.type}")
+        }
+        return listOf()
+    }
+
+    private fun startRegulering() {
+        logger.info("StartReguleringJob startet")
+        rapidsPublisher(
+            UUID.randomUUID(),
+            createRecord(GRUNNBELOEP_REGULERING_DATO),
+        )
+        logger.info("StartReguleringJob ferdig")
+    }
+
+    private fun finnSakerTilRegulering() {
+        logger.info("Finner saker til regulering startet")
+        rapidsPublisher(
+            UUID.randomUUID(),
+            finnSakerTilRegulering(GRUNNBELOEP_REGULERING_DATO),
+        )
+        logger.info("Finner saker til regulering ferdig")
+    }
+}
+
+fun createRecord(dato: LocalDate) =
+    JsonMessage
+        .newMessage(
+            mapOf(
+                ReguleringHendelseType.REGULERING_STARTA.lagParMedEventNameKey(),
+                ReguleringEvents.DATO to dato.toString(),
+                ReguleringEvents.KJOERING to "Regulering-$year",
+                ReguleringEvents.ANTALL to 20,
```

Comment:
TIl senere: Er det en måte vi kan få dette antallet fra et sted vi kan endre det uten git commit? 🤔

Reply:
Kanksje ein eigen databasetabell her med kjoering, antall, spesifikke saker, ekskluderte saker er lurt? Ganske likt den vi hadde for migrering?

Reply:
Ja. Det er ganske mye greiere å lage database instrukser enn kodeendringer som krever bygg og deploy.

Reply:
https://jira.adeo.no/browse/EY-4108